modules/publisher/views.py

```diff
@@ -140,8 +139,6 @@ def publisher_snap_measure(snap_name):
             flask.abort(502, error_messages)
     except ApiResponseError as api_response_error:
         flask.abort(502, str(api_response_error))
-    except ApiError as api_error:
```

Comment:
Should it be removed, or just moved after `MacaroonRefreshRequired`?
So if the error is `ApiError` but is not `MacaroonRefreshRequired` it will still be caught?